### DISCUSSION TITLE
Setup can't find python location

### DISCUSSION THREAD

**Discussion by Keellaeth** [bmaltais/kohya_ss#1548] [Q&A]
I don't know much about coding and in the setup.bat i couldn't find any line for setting up python location. While installing there was the same error and I used "set PYTHON=" code to determine the location but it didn't work here. I saw a discussion about rpvenv.cfg but I don't have any venv folder or cfg like that. It seems like my path doesn't work, I tried different solutions but nothing solved the issue. What should I do?

**Comment by ajellenberger** [comment]
I ran into this problem because ran this before installing Python10. I had Python 11 installed, then I ran "git clone https://github.com/bmaltais/kohya_ss.git" and then encountered a problem when trying to run the setup.bat.

To fix this, I installed the proper version of Python, deleted the kohya folder, restarted my computer, and reinstalled kohya, starting from the beginning. Hope this helps!
